Dockerfile

```diff
@@ -153,21 +145,20 @@ RUN apt-get -y update && \
         gnupg && \
     rm -rf /var/lib/apt/lists/*
 
-COPY etc/lighttpd.conf /lighttpd.conf
-COPY etc/filter-map.lua /filter-map.lua
-COPY etc/400.lua /400.lua
-COPY etc/400-invalid-service-type.lua /400-invalid-service-type.lua
+ADD config/lighttpd.conf /srv/mapserver/config/lighttpd.conf
+ADD config/include.conf /srv/mapserver/config/include.conf
+ADD config/request.lua /srv/mapserver/config/request.lua
 
 RUN chmod o+x /usr/local/bin/mapserv
 RUN apt-get clean
+USER www
 
-
-ENV DEBUG 0
-ENV MIN_PROCS 1
-ENV MAX_PROCS 3
-ENV MAX_LOAD_PER_PROC 4
+ENV DEBUG 5
```

Comment:
is het verstandig om het altijh op debug 5 te hebben?

Reply:
nee dit is aangepast